spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -465,6 +465,22 @@ def provisioning_profiles(mac: false)
       end
     end
 
+    ##
+    # this endpoint is used by Xcode to fetch provisioning profiles.
+    # The response is an xml plist but has the added benefit of containing the appId of each provisioning profile.
+    def provisioning_profiles_via_xcode_api(mac: false)
```

Comment:
I like this design where the client just has a bunch of methods for other callers to use 👍